ci/ReleaseTool/PrepCommand.cs

```diff
@@ -103,13 +88,11 @@ public PrepCommand(Options options)
         public int Run()
         {
             Common.VerifySemanticVersioningFormat(options.Version);
-
-            var remoteUrl = string.Format(Common.RepoUrlTemplate, options.GithubOrgName, options.GitRepoName);
-
+            var gitRepoName = options.GitRepoName;
```

Comment:
Made code between prep and prepfull as similar as possible in naming and variable caching etc.